tests/Loader/GherkinFileLoaderTest.php

```diff
@@ -103,6 +103,7 @@ public function testBasePath(): void
         $this->assertFalse($this->loader->supports('tables.feature'));
         $this->assertTrue($this->loader->supports('features/tables.feature'));
 
+        $this->assertTrue($this->loader->supports('features/pystring.feature'));
```

Comment:
Pretty cool that with the new changes, it actually detects that this test was slightly wrong in principle, since `supports()` should be called with the same argument as when (and before) calling `load()`.